index.Rmd

```diff
@@ -0,0 +1,181 @@
+```
+
+```{r}
+source(here::here("scripts", "functions.R"))
+source(here::here("scripts", "get_data.R"))
+```
+
+```{r}
+births_clean <-
+  births %>% 
+    slice(-c(2,4:55, 57:108, 110:161, 163:214, 216:267, 
+             269:320, 322:373, 375:427, 429:480, 482:533, 
+             535:586, 588:640, 643, 646:698, 700:751, 
+             753:805, 806:858, 860:911, 913:964, 966:1017, 
+             1019:1070, 1072:1176, 1178:1229, 
+             1231:1282, 1284:1391)) %>% 
+  select(-c('1980', '1981', '1982', '1983', '1984', '1985', '1986', '1987', '1988', '2020')) 
+
+births_wide <-
+  births_clean %>%
+  pivot_longer(!Years, names_to = "year", values_to = "children_born") %>% 
+  rename(Type = Years, Year = year, Births = children_born) %>% 
+  select(Year, Births, Type) %>% 
+  mutate(Year = as.numeric(Year)) %>%
+  arrange(Year) %>%
+  mutate(Type = str_replace(string = Type,
+                            pattern = "Less than 15",
+                            replacement = "15 and younger")) %>% 
+  mutate(Type = str_replace(string = Type,
+                            pattern = "\\(People\\)",
+                            replacement = "\\(Births)\\")) %>%
+  pivot_wider(names_from = Type, values_from = Births)
+
+
+
+births_long <- 
+  births_clean %>%
+  
+  pivot_longer(!Years, names_to = "year", values_to = "children_born") %>% 
+  rename(Type = Years, Year = year, Births = children_born) %>% 
+  select(Year, Births, Type) %>% 
+  mutate(Year = as.numeric(Year)) %>% 
+  arrange(Year) %>%
+  mutate(Type = str_remove_all(string = Type,
+                               pattern = "\\(People\\)")) %>% 
+  mutate(Type = str_replace(string = Type,
+                            pattern = "Births ",
+                            replacement = "Total births"
+                            )) %>% 
+  mutate(Type = str_replace(string = Type,
+                            pattern = "Less than 15",
+                            replacement = "15 and younger"))
+```
+
+As we take a look at the following graphs the aim is to discover a trend in birth rates based on the average life expectancy and ethnicity of mothers. Are there clear trends? Do the trends have anything to with each other? Or, are there other elements that could be affecting the trends that we see?
+
+
+# Total births per year
+
+
+First, we take a look at the birth rates for all women in the US. This shows a peak that occurs in 2007 and then a sharp decline in births afterwards. 
+
+```{r}
+birth_graph(total)
+```
+
+
+# Change in birth rates based on the Mother's age
+
+Here we can see that the 35-39 year old age group has a clear increase in birth rates. It crosses paths with the 15-19 year old age group during the year 2000 and continues to increase as the 15-19 year old age group declines. The 20-24 year old age group also mimics the same steady decline that the 15-19 year old age group follows. There is also a uniform peak at all age levels during the year 2007.
+
+```{r}
+birth_graph(age)
```

Comment:
A nice strategy that you might consider here is to utilize a concept called "direct labeling". We didn't discuss this in class, but it's a nice way to improve a visual while reducing the cognitive strain for someone reviewing your graph.